var/spack/repos/builtin/packages/py-ruamel-yaml-clib/package.py

```diff
@@ -14,7 +14,10 @@ class PyRuamelYamlClib(PythonPackage):
     homepage = "https://sourceforge.net/p/ruamel-yaml-clib/code/ci/default/tree/"
     pypi = "ruamel.yaml.clib/ruamel.yaml.clib-0.2.0.tar.gz"
 
+    version("0.2.7", sha256="1f08fd5a2bea9c4180db71678e850b995d2a5f4537be0e94557668cf0f5f9497")
     version("0.2.0", sha256="b66832ea8077d9b3f6e311c4a53d06273db5dc2db6e8a908550f3c14d67e718c")
 
-    depends_on("python@2.7:2.8,3.5:", type=("build", "link", "run"))
+    depends_on("python", type=("build", "link", "run"))
+    # to prevent legacy-install-failure
```

Comment:
Is legacy-install-failure due to python version or setuptools version?

Reply:
It's  the python version. By both using `py-setuptools@67.6.0`, the installation works for `python@3.9.16` but not for `python@3.10.10`